examples/Python/PET/reconstruct_from_listmode.py

```diff
@@ -23,7 +23,7 @@
   -C <cnts>, --counts=<cnts>   account for delay between injection and acquisition start by shifting interval to start when counts exceed given threshold.
   --visualisations             show visualisations
   --nifti                      save output as nifti
-  --gpu                        use gpu
+  --gpu                        use gpu (actually means: use NiftyPET projectors)
```

Comment:
Please keep removed. The STIR listmode recon only supports projmatrixbybin at the moment.

Reply:
do you mean go back to just `use gpu`?